utils/openshift_resource.py

```diff
@@ -113,6 +113,20 @@ def canonicalize(body):
                 body.get('type') == 'Opaque':
             body.pop('type')
 
+        if body['kind'] == 'Route':
+            if 'status' in body:
+                body.pop('status')
+            if body['spec'].get('wildcardPolicy') == 'None':
```

Comment:
This means that if wildcardPolicy is not explicitely set, it will be be set to `None`, right?

Reply:
I mean, that's what k8s will do, no?

Reply:
right. same as ConfigMap's type `Opaque`.